src/wasm-traversal.h

```diff
@@ -49,6 +49,7 @@ struct Visitor {
   ReturnType visitSetGlobal(SetGlobal* curr) {}
   ReturnType visitLoad(Load* curr) {}
   ReturnType visitStore(Store* curr) {}
+  ReturnType visitAtomicRMW(AtomicRMW* curr) {return ReturnType();} //Stub impl so not every pass has to implement this yet.
```

Comment:
Right, makes sense. Yeah, I think we should update them all to the `{ return ReturnType; }` form